pkg/bucketclass/bucketclass.go

```diff
@@ -176,6 +177,7 @@ func CmdReconcile() *cobra.Command {
 		Use:    "reconcile",
 		Short:  "Runs a reconcile attempt like noobaa-operator",
 		Run:    RunReconcile,
```

Comment:
same as for backingstore - reconcile expects args

Reply:
10x